lib/Search/Provider.php

```diff
@@ -71,6 +71,11 @@ public function getName(): string {
 	}
 
 	public function getOrder(string $route, array $routeParameters): int {
+		if (strpos($route, Application::APP_ID . '.') === 0) {
```

Comment:
@StCyr please attach a debugger here so we can get the value of ``$route`` here

Reply:
![image](https://user-images.githubusercontent.com/8179031/98655600-ef526b80-233f-11eb-9a69-40d051709794.png)

Reply:
:open_mouth: oh

What is the full URL sent (without domain)?

Reply:
http://192.168.0.128/nextcloud/ocs/v2.php/search/providers?from=/nextcloud/apps/mail/box/12/thread/1063

Reply:
It's likely an issue because your nextcloud is hosted in a subdirectory. You'll experience the same if you have Talk installed. The search won't show Talk results on top, while it does with installations that have Nextcloud running in the root directory.

Reply:
yeah, apparently, this occurs here: 

	public function findMatchingRoute(string $url): array {
https://github.com/nextcloud/server/blob/769f59ec657a4dcb993550ef33341015d5ea7942/lib/private/Route/Router.php#L245

Reply:
Thanks for looking into it and opening the ticket!